README.md

```diff
@@ -32,7 +32,9 @@ async fn main() {
         }
     };
 
-    pin_mut!(s); // needed for iteration
+    // Stream must be pinned before iteration. It can be done with Box::pin,
+    // but the pin_mut macro is more efficient. See "Pinning" section.
+    pin_mut!(s);
```

Comment:
minor nit, could we possibly change this to https://docs.rs/tokio/0.2.11/tokio/macro.pin.html happy to do this in another PR but would be good to point to the tokio version.